src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java

```diff
@@ -84,7 +86,13 @@
  *
  *      for (;;){}               // OK
  *      for(;;){}                // violation, space after 'for' is required
- *      }
+ *
+ *      try (InputStream ignored = System.in) {} // OK
+ *      try(InputStream ignored = System.in) {} // violation ''try' is not followed by whitespace'
+ *
+ *      try {} catch(Exception exception){} // OK
```

Comment:
`exception` -> `e`

Reply:
also, please add a space after catch (below line, too), this will become a violation after we add `LITERAL_CATCH` token, and we might forget to update examples.

Reply:
https://github.com/checkstyle/checkstyle/issues/11498

Reply:
Changes Done

Reply:
Resolved.